schema/crdb/dbinit.sql

```diff
@@ -216,37 +216,26 @@ CREATE INDEX IF NOT EXISTS lookup_sled_by_policy_and_state ON omicron.public.sle
     sled_state
 );
 
-CREATE TYPE IF NOT EXISTS omicron.public.sled_resource_kind AS ENUM (
-    -- omicron.public.vmm ; this is called "instance" for historical reasons.
-    'instance'
-    -- We expect to other resource kinds here in the future; e.g., to track
-    -- resources used by control plane services. For now, we only track
-    -- instances.
-);
-
--- Accounting for programs using resources on a sled
-CREATE TABLE IF NOT EXISTS omicron.public.sled_resource (
-    -- Should match the UUID of the corresponding resource
+-- Accounting for VMMs using resources on a sled
+CREATE TABLE IF NOT EXISTS omicron.public.sled_resource_vmm (
+    -- Should match the UUID of the corresponding VMM
     id UUID PRIMARY KEY,
```

Comment:
Nitpicky, feel free to ignore me: maybe we should change this to `vmm_id` for clarity while we're messing with this table?

Reply:
If it's cool with you, I'm going to punt on this, because https://github.com/oxidecomputer/omicron/blob/ae63a611670b4bc4c8486dbf82f26e89e8ed7866/schema/crdb/README.adoc?plain=1#L201-L212

I can't really do a simple column rename here, I would need to perform a data migration which would also move the UUID to the end of the table.

TL;DR, this is possible, just... painful

Reply:
Ugh, yeah, I don't think that's worth the effort. Carry on.